osu.Game/Overlays/Dashboard/CurrentlyPlayingDisplay.cs

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Collections.Specialized;
+using System.Linq;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Screens;
+using osu.Game.Online.API;
+using osu.Game.Online.API.Requests;
+using osu.Game.Online.Spectator;
+using osu.Game.Screens.Play;
+using osu.Game.Users;
+using osuTK;
+
+namespace osu.Game.Overlays.Dashboard
+{
+    internal class CurrentlyPlayingDisplay : CompositeDrawable
+    {
+        private IBindableList<int> playingUsers;
+
+        private FillFlowContainer<UserPanel> userFlow;
+
+        [Resolved]
+        private SpectatorStreamingClient spectatorStreaming { get; set; }
+
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            RelativeSizeAxes = Axes.X;
+            AutoSizeAxes = Axes.Y;
+
+            InternalChild = userFlow = new FillFlowContainer<UserPanel>
+            {
+                RelativeSizeAxes = Axes.X,
+                AutoSizeAxes = Axes.Y,
+                Spacing = new Vector2(10),
+            };
+        }
+
+        [Resolved]
+        private IAPIProvider api { get; set; }
+
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+
+            playingUsers = spectatorStreaming.PlayingUsers.GetBoundCopy();
```

Comment:
Not necessarily a problem with this PR specifically but for some reason the collection change callbacks on this get called every now and again, which makes this display pretty hard to use before it's fully populated as things move around a lot. Probably going to need to be looked at server-side but figured I'd say it just in case.

Reply:
The issue is we don't have an API endpoint to lookup multiple users at once right now, so the API responses come in one at a time. Need this to be fixed before deploying this new screen.